sentences/en/_common.yaml

```
@@ -74,9 +74,11 @@ lists:
         out: "fahrenheit"
   brightness_level:
     values:
-      - in: (max|maximum|highest)
+      - in: ([the] (max|maximum|highest) [(level|value|setting|brightness)]|full [(brightness|light)]|all the way up|as (bright|light) as possible)
         out: 100
-      - in: (min|minimum|lowest)
+      - in: "[the] (half|halfway|mid) [(level|setting|brightness|bright|light|lit)]"
```

"The mid lit" doesn't sound right
```suggestion
      - in: "([the] (half|halfway|mid) [(level|setting|brightness|light)]|(half|halfway|mid) (bright|lit)"
```